### PR TITLE
Callout CIS Ref Arch

### PR DESCRIPTION
**Pull request by yorinasub17**
We are very close to having the CIS Ref Arch ready for launch (https://github.com/gruntwork-io/cis-compliance-aws/pull/16 and https://github.com/gruntwork-io/usage-patterns/pull/356 are the two remaining PRs for the must have items from https://github.com/gruntwork-io/team-product/pull/7), so decided to open this PR to tee up the marketing activities for launching the offering.

### CODE REVIEW

File: _data/cis-how-it-works.yml [hunk 1]
```diff
@@ -21,6 +21,12 @@
       <li>Avoid overly permissive inbound security group rules</li>
     </ul>
 
+    <p class="alert" style="border: 1px solid #194c5f; background-color: #242e3b;">
+      <strong>Get an End to End Production-Grade Architecture</strong><br>
```
Comment (brikis98) [inline _data/cis-how-it-works.yml]:
```suggestion
      <strong>Get an End-to-End CIS Compliant Production-Grade Architecture</strong><br>
```

File: _data/cis-how-it-works.yml [hunk 2]
```diff
@@ -21,6 +21,12 @@
...
+      Request a <a href="/reference-architecture">Gruntwork Reference Architecture</a> to get an end to end
+      production-grade CIS Benchmark compliant architecture in 1 day.
```
Comment (brikis98) [inline _data/cis-how-it-works.yml]:
```suggestion
      production-grade architecture that has been certified by Gruntwork and CIS to be compliant with the CIS Benchmarks, deployed into your AWS accounts, and fully managed as code—all in about 1 day!
```

File: pages/checkout/_add-ons.html [hunk 3]
```diff
@@ -41,6 +41,10 @@ <h1 class="addon-amount" id="addon-amount-refarch">$4,950</h1>
             <h3 class="addon-type">CIS Benchmark Compliance</h3>
             <hr class="hr-sm hr-reset visible-sm-block visible-md-block visible-lg-block">
             <p>Achieve and maintain compliance with the CIS AWS Foundations Benchmark.</p>
+            <p class="text-muted small text-center margin-bottom-none visible-sm-block visible-md-block visible-lg-block">
+              Add <a href="#reference-architecture">Reference Architecture</a> to get a CIS compliant end to end
+              production-grade AWS architecture in 1 day.
```
Comment (brikis98) [inline pages/checkout/_add-ons.html]: See above

Comment (yorinasub17) [reply pages/checkout/_add-ons.html]: Updated abdb435 

But not sure if it looks good with the longer text

Customize_your_plan_with_add-ons

Thoughts?

Comment (bwhaley) [reply pages/checkout/_add-ons.html]: I'd consider phrasing it as follows:

> Achieve and maintain compliance with the CIS AWS Foundations Benchmark. Our code is certified compliant by the Center for Internet Security. Add the Reference Architecture to get an end-to-end, production-grade architecture in your own AWS environment, fully managed as code - all in about 1 day!

I'd also leave it as one paragraph with the same font.

File: _data/cis-how-it-works.yml [hunk 4]
```diff
@@ -21,6 +21,12 @@
       <li>Avoid overly permissive inbound security group rules</li>
     </ul>
 
+    <p class="alert" style="border: 1px solid #194c5f; background-color: #242e3b;">
+      <strong>Get an End-to-End CIS Compliant Production-Grade Architecture</strong><br>
+      Request a <a href="/reference-architecture">Gruntwork Reference Architecture</a> to get an end to end
+      production-grade architecture that has been certified by Gruntwork and CIS to be compliant with the CIS Benchmarks, deployed into your AWS accounts, and fully managed as code—all in about 1 day!
```
Comment (bwhaley) [inline _data/cis-how-it-works.yml]:
```suggestion
      production-grade environment, certified by CIS for the AWS Foundations Benchmark, deployed into your AWS accounts, and fully managed as code—all in about 1 day!
```

File: pages/checkout/_add-ons.html (hunk 3 above)
Comment (yorinasub17) [reply pages/checkout/_add-ons.html]: Done! 3c2e40d 

Making it one paragraph does look better:

Customize_your_plan_with_add-ons

**Comment by yorinasub17** [comment]
@brikis98 FYI I no longer have access to deploy the website (I am not one of the approved users to the prod dogfood account), so if you want to push this live before the holidays, you will have to do it.

**Comment by brikis98** [comment]
Roger! I will merge and deploy now.

**Comment by brikis98** [comment]
Deployed!

**Pull request merged**